wolfcrypt/src/asn.c

```diff
@@ -17470,6 +17474,9 @@ static int DecodeResponseData(byte* source,
             }
             single = single->next;
             XMEMSET(single, 0, sizeof(OcspEntry));
+            single->status = (CertStatus*)XMALLOC(sizeof(CertStatus),
```

Comment:
Thank you @bernd-edlinger for pointing out the malloc is not checked for NULL. 
This should have been caught in the multiple peer reviews....

I do not see this fixed in master or a pending PR. @ejohnstown or @guidovranken please put up a PR to fix.

Thanks, David Garske, wolfSSL

Reply:
Fix: https://github.com/wolfSSL/wolfssl/pull/4093